src/renderer/fonts/font_loader.rs

```diff
@@ -46,36 +46,46 @@ pub struct FontLoader {
 }
 
 #[derive(Debug, Hash, PartialEq, Eq, Clone)]
-pub enum FontKey {
+pub struct FontKey {
+    // TODO(smolck): Could make these private and add constructor method(s)?
+    // Would theoretically make things safer I guess, but not sure . . .
+    pub bold: bool,
+    pub italic: bool,
```

Comment:
Thoughts on the TODO here?

Reply:
eh. I don't really care that much. Its only us consuming these types atm so I'd just leave the todo and move on. Looks good